tools/sof-kernel-log-check.sh

```diff
@@ -310,6 +310,11 @@ ignore_str="$ignore_str"'|proc_thermal 0000:00:..\..: No auxiliary DTSs enabled'
 # elan_i2c i2c-ELAN0000:00: invalid report id data (ff)
 ignore_str="$ignore_str"'|elan_i2c i2c-ELAN0000:.*: invalid report id data'
 
+# SoundWire workaround using RT701
+# https://github.com/thesofproject/linux/pull/2701
+ignore_str="$ignore_str"'|rt711 sdw:[0-3]:25d:701:0: Probe of rt711 failed: -19'
+ignore_str="$ignore_str"'|rt1308 sdw:[0-3]:25d:701:0: Probe of rt1308 failed: -19'
+
```

Comment:
Can you guard this with:
```shell
if [ "$platform" = xxx ] ; then
  ignore_str="$ignore_str"'|rt711 ...'
  ignore_str="$ignore_str"'|rt1308 ...'
fi 
```
The PR URL is super useful but... very long! Can you also add just a couple words like:
```shell
# some BIOSes are  buggy so we need to probe like mad.
```
This helps a lot with future maintenance of this file.